site/src/pages/WorkspacePage/WorkspaceScheduleControls.test.tsx

```diff
@@ -1,49 +1,28 @@
 import { render, screen } from "@testing-library/react";
-import { ThemeProvider } from "contexts/ThemeProvider";
-import { QueryClient, QueryClientProvider, useQuery } from "react-query";
-import { MockWorkspace } from "testHelpers/entities";
-import { WorkspaceScheduleControls } from "./WorkspaceScheduleControls";
-import { workspaceByOwnerAndName } from "api/queries/workspaces";
+import { type FC } from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
 import { RouterProvider, createMemoryRouter } from "react-router-dom";
 import userEvent from "@testing-library/user-event";
-import { server } from "testHelpers/server";
-import { rest } from "msw";
 import dayjs from "dayjs";
 import * as API from "api/api";
 import { GlobalSnackbar } from "components/GlobalSnackbar/GlobalSnackbar";
+import { ThemeProvider } from "contexts/ThemeProvider";
+import { MockTemplate, MockWorkspace } from "testHelpers/entities";
+import { WorkspaceScheduleControls } from "./WorkspaceScheduleControls";
 
-const Wrapper = () => {
-  const { data: workspace } = useQuery(
-    workspaceByOwnerAndName(MockWorkspace.owner_name, MockWorkspace.name),
+const Wrapper: FC = () => {
```

Comment:
Maybe we could remove the `Wrapper` component and mount the component directly in the render function.

Reply:
I actually had to revert these because I didn't really get how the tests worked 😂